package.json

```diff
@@ -44,11 +44,12 @@
     "enzyme-adapter-react-16": "^1.14.0",
     "eslint-config-airbnb": "^17.1.1",
     "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-jest": "^22.7.2",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.14.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react": "^7.14.3",
+    "glob-gitignore": "^1.0.14",
     "prettier": "^1.18.2"
   }
```

Comment:
What do you think for the pre-commit check? 
https://prettier.io/docs/en/precommit.html#option-2-pretty-quick-https-githubcom-azz-pretty-quick

Reply:
I think it's a good idea. Can you implement it?

Reply:
Sure, let’s merge your pr then I will create a PR for that.